### ISSUE TITLE
Models with I18n behavior does not work correctly

### ISSUE THREAD

**Issue by fruit** [fruit/sfCacheTaggingPlugin#9]
When updating `i18n-table` column, does not increases object`s tag version.

**Comment by fruit** [comment]
In `preSave()` for `i18n` model, `$modifiedColumns` are always an empty array

```
  if (! $object->isNew() && 0 == count($modifiedColumns))
  {
    return;
  }
```

**Comment by fruit** [comment]
I18n behavior closed by 8660124f82f62b22ba5c26f4ab67ccc3858f3cef, added tests on getTags() closed by 8660124f82f62b22ba5c26f4ab67ccc3858f3cef
